examples/bidder/http_bidder_test.cpp

```diff
@@ -95,49 +103,63 @@ int main(int argc, char *argv[]) {
         LOG(error) << e.what();
         return 0;
     }
-    exchange_handler<DSL::GenericDSL<>> bid_handler(std::chrono::milliseconds(config.data().timeout));
+    
+
+    using bid_handler_type = exchange_handler<DSL::GenericDSL<>, vanilla::VanillaRequest>;
```

Comment:
Year the problem in in vanilla::BidRequest and VanillaRequest.
They are not flexible